kotlin-android/app/src/main/java/com/mongodb/tasktracker/ProjectActivity.kt

```diff
@@ -0,0 +1,114 @@
+package com.mongodb.tasktracker
+
+import android.app.AlertDialog
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.EditText
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import com.mongodb.tasktracker.model.Project
+import com.mongodb.tasktracker.model.ProjectAdapter
+import com.mongodb.tasktracker.model.User
+import io.realm.Realm
+import io.realm.RealmList
+import io.realm.kotlin.where
+import io.realm.mongodb.sync.SyncConfiguration
+import java.util.*
+
+class ProjectActivity : AppCompatActivity() {
+    private var user: io.realm.mongodb.User? = null
+    private lateinit var realm: Realm
```

Comment:
It might be clearer to rename this to `userRealm`, if that's what it is

Reply:
On the fence about renaming the realm member variables for each activity. On one hand, it could make it easier to understand the difference between each. On the other, it could be more confusing for readers who think there are different kinds of realm instead of just different partitions? Maybe userPartition? But we do want to keep the word ``realm`` in there... suggestions welcome!

Reply:
I think it is fine to call it `userRealm`, etc. It's the realm opened with a user in it.